src/components/tooltip/Marker.js

```diff
@@ -139,7 +140,10 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
         if (innerWindowIDSuffix) {
           url += innerWindowIDSuffix;
         }
-        return <TooltipDetail label="URL">{url}</TooltipDetail>;
+        if (page.isPrivateBrowsing) {
+          url += ' (private)';
+        }
+        return <TooltipDetail label="Page">{url}</TooltipDetail>;
```

Comment:
The change from URL => Page here is the only thing that I added in the commits you already reviewed. I missed it previously (I properly had the change above).